folly/io/async/AsyncUDPSocket.cpp

```diff
@@ -36,6 +36,14 @@
 #define SO_REUSEPORT 15
 #endif
 
+#ifndef IPV6_TCLASS
+#if defined(__GNU__)
+#define IPV6_TCLASS 61
```

Comment:
Where was this value gotten from?

Reply:
Linux headers: https://github.com/torvalds/linux/blob/master/include/uapi/linux/in6.h#L259
Should I drop it?